src/demos/textarea.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>TextArea</title>
+    <style>
+      .parent {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: row;
+        color: var(--calcite-ui-text-3);
+        font-family: var(--calcite-sans-family);
+        font-size: var(--calcite-font-size-0);
+        font-weight: var(--calcite-font-weight-medium);
+        padding: 15px;
+        margin: 0 7% 0 7%;
+      }
+
+      .child {
+        display: flex;
+        flex-direction: column;
+        flex-flow: column;
+        margin: 15px;
+        padding: 0 18px;
+      }
+
+      .scale-text {
+        flex-basis: 320px;
+      }
+
+      .child-text {
+        flex-direction: column;
+        flex-flow: column;
+        display: inline-flex;
+        flex-basis: 160px;
+        padding: 0 10px;
+        justify-content: center;
+      }
+
+      calcite-textarea {
+        width: 320px;
+        height: 160px;
+      }
+    </style>
+    <script src="./_assets/head.js"></script>
+  </head>
+  <body>
```

Comment:
Can you add the `demo-dom-swapper` custom element like the rest of the demos have